## Introduction 

Nucleotide excision repair (excision repair) is an essential mechanism to remove bulky DNA adducts including UV-induced DNA lesions [@doi:10.1074/jbc.R117.807453]. 
As in other repair systems, excision repair starts with damage recognition. 
There are two subpathways in damage recognition step leading to two repair mechanisms: global repair (GR) and transcription-coupled repair (TCR). 
GR is active throughout the genome whereas TCR is only active on the transcribed strands as it benefits from the efficient recongition of the stalled RNA polymerase II (RNAPII). 

To reveal genome-wide excision repair dynamics, heterogeneity and associations, eXcsion repair sequencing (XR-seq) method was developed.
It was found that TCR is more efficient particularly for slowly-repaired DNA lesions. 
For instance, among UV photoproducts 6-4 pyrimidine-pyrimidone ([6-4]PP) and cyclobutane pyrimidine dimer (CPD), CPD is more prone to TCR as it is less efficiently repaired by GR. 
Although DNA lesions are biasedly formed at certain regions in the genome, the heterogeneity of repair is majorly due to the repair bias [@doi:10.1073/pnas.1706522114; @doi:10.1038/s41467-018-05064-0]. 
Repair bias is mostly affected by transcription and chromatin [[@doi:10.1101/gad.261271.115; @doi:10.1073/pnas.1614430113; @doi:10.1073/pnas.1603388113; @doi:10.1038/s41467-019-08290-2].

To date, genome-wide repair maps were generated for model organisms including *Escherichia coli* [@doi:10.1073/pnas.1700230114; @doi:10.1074/jbc.C117.818807], *Saccharomyces cerevisia* [@doi:10.1073/pnas.1801687115], *Drosophila melanogaster* [@doi:10.1074/jbc.AC119.011448], *Arabidopsis thaliana* [@doi:10.1038/s41467-018-03922-5], *Mus musculus* [@doi:10.1073/pnas.1804493115], and *Homo sapiens* [@doi:10.1101/gad.261271.115; @doi:10.1073/pnas.1706522114]. 
TCR presence was verified for each of these organisms. 
For eukaryotic genomes, the consistent finding was the efficient repair in open chromatin regions.
Heterochromatin regions were found to be repaired at later time points.
Human repair profiles were extensively studied with respect to damage formation and chromatin states.
Whether regions in human genome that are efficient with respect repair are specific to human or not is yet to be studied.

To study whether the features of overall repair is unique to the organism of interest we aimed to compare human and a non-human primate. 
Gray mouse lemur (*Microcebus murinus*) stands out as a promising model organism candidate because of its small body size, short gestation time (2 months) and fast sexual maturation (6-8 months) [@doi:10.1534/genetics.116.199448].  
A mouse lemur's genome was recently sequenced and assembled [@doi:10.1186/s12915-017-0439-6]. 
With no surprise, it was shown that mouse lemur and human orthologs share ~91% identity. 
Although their genome is now available, they still lack an in-depth understanding of genomic features such as epigenetic maps, transcriptomes, methylomes etc.

Derived from the common ancestor of primates, same cell types from human and mouse lemur might behave similary in response to DNA damage.
With this motivation, we carried out a comparative study between human and mouse lemur to understand similarities and differences between their repair profiles.
We derived primary fibroblasts from mouse lemur and immortalized them.
From this cell line, we generated transcriptomes with RNA-seq.
We exposed cells to UV and performed XR-seq.
XR-seq captured the excised oligomers as repair products for two main UV-induced damage types: (6-4)PP and CPD.
We compared lemur and human fibroblast XR-seq datasets with respect to repair efficiency.
