## Results

### Excised oligomer characteristics

In vivo excision assay resulted in the excised oligomers containing UV-induced lesions. (Fig {@fig:fig1}A)
Excised oligomers were captured for two damage types distinctly with specific anti-damage antibodies. 
The oligomer length distribution varies from 16 to 30. 
Two distinct intense bands were observed, which indicate primary and degraded excised oligomers. 
The gel images show that the primary excised oligomer lengths vary between 23-25nt.
Through time course, gel images reflect a more intense secondary product band, suggesting higher levels of degraded excised oligomer with damage is observed at later time points. 
For human (NHF1) and mouse lemur, this trend is similar. 

Nucleotide excision repair in human, yeast and *E. coli* was shown to yield primary excised oligomers predominantly in lengths of 27, 24 and 13 nucleotides, respectively [@doi:10.1074/jbc.M113.482257; @doi:10.1073/pnas.89.8.3664; @doi:10.1073/pnas.1801687115; @pmid:6380755; @doi:10.1073/pnas.1700230114]. 
As a primate, mouse lemur resulted in excised oligomer distribution identical to humans (Fig {@fig:fig1}B). 
An identical pattern of oligomer length distribution between human and mouse lemur is independent of the damage type. 
For both CPD and (6-4)PP, nucleotide excision repair machinery incised 26nt oligomer as the predominant product. 
A small variance in oligomer lengths between in vivo excision assay and XR-seq might be due to the immediate exonuclease activity on excised oligomer for in vivo excision assay. 
Because of the TFIIH coimmunoprecipitation in excision repair sequencing technique, XR-seq primary products are less prone to degradation, possibly because of TFIIH protection of the excised oligomers.

### Identical CPD and (6-4)PP nucleotide frequency between human and lemur

Nucleotide frequencies of the excised oligomers between human and mouse lemur are identical for both CPD and (6-4)PP (Fig {@fig:fig1}C). 
Pyrimidine enrichment was revealed at 19th, 20th and 21st nucleotides for CPD and (6-4)PP for both organisms, which is in agreement with the incision site at 6 to 8 nt fixed distance to the 3' end of the DNA lesion. 
The nucleotide contents for CPD and (6-4)PP are different, as they were previously reported and discussed [@doi:10.1101/gad.261271.115].

![ Characteristics of excised oligomers upon repair of UV-induced damages. 
*(A)* In vivo excision assay for (6-4)PP and CPD are shown. 
*(B)* Length distribution of excised oligomers for CPD (top) and (6-4)PP (bottom). Human and mouse lemur are shown in blue and red, respectively. 
*(C)* Nucleotide frequency in the predominant oligomer (26 nt) for each primate and damage type. Representative data from replicate 1.
](images/fig-1.png){#fig:fig1 .white}

### UV repair efficiency of mouse lemur in comparison with human

To identify the overal repair efficieny upon UV irradiation we performed a survival assay. 
We seeded 300 cells and grew them for 16 hours. 
Cells were exposed to UV-C at 1 Joule m^2/sec for variable time duration to reach indicated doses from in the range of 2.5 to 10 J/m^2.
We counted cells for each dose and plotted a survival curve for lemur and human fibroblasts (Fig {@fig:fig2}A).
Based on the survival curves we infer that lemur fibroblasts are more sensitive to UV relative to human cells. 
This suggests a differential repair rates between two organisms. 

We than performed immunoslot blot assays (Fig {@fig:fig2}B).
Cells were exposed to UV-C at 10J/m^2. 
To separate UV-induced DNA lesions we used specific antibodies for CPD and (6-4)PP. 
We quantified the remaining damage signal for each damage type for both organisms. 
Both (6-4)PP and CPD repair appeared to be more efficient in human cells compared to mouse lemur. 
In 2 hours (6-4)PP repair was completed in humans wehereas in lemurs, it took 4 hours to complete (6-4)PP repair. 
After 32 hours of irradiation 20% of CPDs were still unrepaired for human cells. 
This ratio was more than 40% for mouse lemurs. 
Based on the differencies of the organismal curves, the major difference between two cell lines was due to their differential CPD repair efficiencies.
Survival and immonoslot blot assays are in harmony that they suggest mouse lemur cells have relatively low repair rate in comparison with human NHF1 cells.

![ UV sensitivity and repair rates in human and lemur fibroblasts. 
*(A)* Cell plates were shown after indicated UV dosage (left) and cells are quantified (right).
*(B)* Immunoslot blot assays for both organisms and each damage type (left).
Remaining damage levels at each time point were quantified and plotted (right).
](images/fig-2.png){#fig:fig2 width=66% .white}

### Prominent transcription-coupled repair in mouse lemur

Nucleotide excision repair comprises two sub-pathways each of which distinctly acts on damage recognition. 
Global repair is the genome-wide mechanism that is active in any region in the genome, although its efficiency depends on chromatin factors. 
On the other hand, transcription-coupled repair (TCR) depends on the RNA polymerase II stalled at the lesion. 
Stalled RNAPII recruits transcription-coupled repair factor, CSB, to enhance repair. 
As TCR occurs when RNAPII is stalled, it is active in the regions that are transcribed by this polymerase. 
Also, TCR is only seen in the transcribed strand of the genes. 
DNA lesions on the non-transcribed strand are subject to global repair. 
A major rate-limiting step of nucleotide excision repair is damage recognition. 
Lesions are recognized more efficiently when they stall, otherwise progressing RNAPII. 
For this reason, for the organisms having TCR, genes have asymmetrical repair between two strands; TS is repaired more efficiently compared to NTS. 

(6-4)PP is known to be repaired faster than CPD [@doi:10.1093/nar/gkz035; @doi:10.1073/pnas.1603388113]. 
Therefore, global repair for (6-4)PP is highly active and there is no need for TCR advantage in damage recognition. 
(6-4)PP strand asymmetry between TS and NTS (TS/NTS ratio) is much weaker compared to the strand asymmetry of CPD repair [@doi:10.1073/pnas.1603388113].
CPD, on the other hand, is harder to repair and it takes much longer time to remove CPDs in the genome. 
For this reason, at early time points (such as 1 hour after UV irradiation), we observe a strong TCR effect which is evaluated by the asymmetrical repair between TS and NTS. 
Here, we compared the genic strand asymmetry between human and mouse lemur (Fig {@fig:fig3}). 
We retrieved all annotated protein-coding genes from both genomes. 
We removed the genes that are closer to each other with less than 20kb in order ti remove the signal that might have a "canceling-out" effect. 
We used 5277 and 3366 annotated genes for human and mouse lemur, respectively (see methods). 
We performed a meta-analysis where we aligned all transcription start and transcription end sites and calculated the RPKM (reads per kilobase per million mapped reads) values.
Interestingly, mouse lemur fibroblasts exhibited stronger TCR profiles compared to human cell lines. 

There is an apparent asymmetry between both human and lemur TCR profiles relative to TSS (Fig {@fig:fig3}). 
The reason for this was suggested because of the bi-directional promoter features of the promoters [@doi:10.1016/j.tig.2011.04.002].

![ Transcription-coupled repair in mouse lemur and human fibroblasts. Transcription start sites (TSS) and transcription end sites (TES) were retrieved from GTF files for two genomes. 
10 kb upstream and downstream of TSS and TES were divided into 100 bp windows. 
XR-seq reads falling onto each bin were normalized to RPKM. 
Opposite strand mappings (gene:XR-seq pair as +:- or -:+) are annotated as TS (red), same strand mappings (+:+; -:-) were annotated as NTS. 
The XR-seq RPKM values were normalized by the shuffled RPKM values derived by mapped reads onto sites which are random genomic sites. 
CPD and (6-4)PP repair events were shown at the top and bottom, respectively. 
Only replicate 1 is shown.
](images/fig-3.png){#fig:fig3 .white}


### Conserved repair rates between homologous regions of two primates

In order to compare and contrast the genome-wide repair profiles between humans and mouse lemurs, we retrieved the homologous genomic regions between the organisms (Fig {@fig:fig4}A). 
We aligned two genomes and kept the regions that have at least 80% similarity.
Due to the low coverage of the XR-seq samples we removed the aligned segments that are shorter than 400 bases to eliminate the randomn effect due to scarce repair events. 
Lineage specific genome duplications result in paralogs that can be co-orthologs to one locus in the other genome. 
This situation is known as one-to-multiple othology relationship. 
As we cannot be sure which one of the multiple paralogs in one genome is the "true" ortholog of a region in the other one, we filtered out one-to-multi kinds of homologous relationships and kept one-to-one holomogs only.

1.0% and 1.3% of the genomes were taken into account with one-to-one homologs for human and lemur genomes, respectively.
4-5% and 2-3% of the XR-seq reads were mapped to those regions for CPD and (6-4)PP, respectively.
The reason why mapping rate of CPD is higher than (6-4)PP is due to its slower repair rate.
Slowly repaired CPDs are more prone to the effects of chromatin. 
Because we applied stringent criteria to retrieve the homologous regions with at least 80% identity, they are likely to be more open compared to other regions in the genome.
Therefore, CPD repair events are mapped onto those regions more than (6-4)PP reads did.

We investigated the correlation between the repair profiles of the two organisms. 
Although there is a clear difference in TCR rates, homologous regions of two genomes exhibited a strong repair rate correlation.

![ **Conserved repair rates between human and mouse lemur.**
](images/fig4.png){#fig:fig4 .white}

### Repair profile association is correlated with gene expression

To understand what factors affect the correlation between two organisms, we questioned the correlation strength and expression levels. 
We prepared RNA-seq data sets for mouse lemur fibroblasts and obtained RNA-seq datasets for human fibroblasts from a publicly available database (see methods). 
The strenght of the correlation between two repair profiles were found to be correlated with transcription strength.
Highly expressed regions had better repair correlation between two organisms.

We further investigated whether evolutionary conservation is a factor in the repair correlation. 
To do that we checked whether the correlation strength is correlated with the percent identity of the homologous regions. 
The results showed that the highly conserved regions between two organisms had stronger repair level correlation. 

![ **Repair rate consistency in correlation with expression and evolutionary conservation.**
](images/fig5.png){#fig:fig5 .white}

### Same cells from two primates have more similar repair profiles relative to two different cells from the same organism

The correlation of repair profiles between two primates might simply be due to the mappability of the genomes. 
Such a correlation makes sense only when we measure the similarity of repair patterns relative to another dataset. 
For this reason, we used another human cell line, GM12878 - B-Lymphocytes to see whether the repair profile of human fibroblast is closer to lemur fibroblast or human lymphocyte.
We performed XR-seq for CPD only as it yields higher genome-wide heterogeneity probably because it is slowed at a much slower rate compared to (6-4)PP.
Human and mouse lemur fibroblasts clustered together in the PCA plot (Fig {@fig:fig6}). 
Within fibroblasts, biological replicates were grouped together. 
More interestingly, (6-4)PP and CPD samples for separated groups each of which has human and lemur samples.
On the other hand, CPD XR-seq samples of human lymphocytes were clustered as an out group.
This result suggest that human and mouse lemur fibroblasts have similar chromatin and expression patterns and therefore have similar repair profiles. 
On the other hand, belonging to the same organisms, completely different human cell lines have a great variation with respect to repair patterns.
It is also interesting that although TCR profiles between two primates are quite different than each other, they share high repair profile similarity. 

![ **Cell type-based consistency of repair profiles between two primates.**
](images/fig6.png){#fig:fig6 width=100% .white}