## Methods

### Fibroblast cell line derivation

### XR-seq library preparation

### RNA-seq library preparation

### Human and lemur alignment

MUMmer version 3.23 [@doi:10.1186/gb-2004-5-2-r12] was used to align grey mouse lemur genome (Mmur_3.0) to human genome (hg19) with `nucmer` subprogram.
The alignments were filtered based on criteria taking length and identity into account.
Alignments are listed in “delta format” by default by Nucmer.
For every reference (human) - query (lemur) pair, we kept the alignments which form the longest mutually consistent set.
Filtering step was performed with `delta-filter` subprogram.
`show-coords` subprogram was used to display summary information such as position, percent identity and other features of each alignment, in Btab format with `-B` and `-rclo` arguements.

We used `generateBED.R` [@url:https://github.com/compGenomeLab/lemurRepair] to write homologous human-lemur regions in a bed file.
Due to low genome coverage in XR-seq data sets small regions introduce random repair values.
Since we are not interested in too short alignments, we filtered out alignments shorter than 400 bp while generating the bed file.
Additionally, alignments with at percent similarity lower than 80 are filtered out of the list.
Resulting is a BED file with each homologus region between two genomes.

This output file contains overlapping regions; one region in either genome might be aligned to more than one region in the other genome.
To remove multi-mapping regions, we used `bedtools intersect` [@doi:10.1093/bioinformatics/btq033] to intersect the file with itself.
The command line we used was: `bedtools intersect -wo -s -a humanOverlapsLemur_short_noFilter.bed -b humanOverlapsLemur_short_noFilter.bed > human_intersect.txt`.
We compared this intersected output file to the `bed` file generated previously using `findDupAln.R` [@url:https://github.com/compGenomeLab/lemurRepair] to exclude regions in both genomes that align to multiple regions in other genome.
This resulted in a `bed` file of "one-to-one" homologous regions without multiple alignments for one region.


### XR-seq analysis

We trimmed 3’ adapter using Cutadapt [@doi:10.14806/ej.17.1.200].
Bowtie2 version 2.3.4.1 was used to align sequencing reads to reference genomes.
With a two-step conversion process, the `.sam` format is first converted (with SAMtools [@doi:10.1093/bioinformatics/btp352]) to `.bam` format, which is then converted to `.bed` format with BEDTools [@doi:10.1093/bioinformatics/btq033].
BED file is sorted by coordinates by removing duplicates.

### RNA-seq analysis

Sequencing reads were aligned to reference genome using STAR Aligner 2.6.1a [@doi:10.1093/bioinformatics/bts635] with default parameters.
The aligned reads are converted to `.bed` format with `bedtools bamtobed`.
The resulting `.bed` files are sorted by coordinates and duplicate regions are removed.

### XR-seq simulation

We evaluate the correlation of repair mechanisms on both simulated and real XR-seq datasets.
For this reason, a simulated dataset is generated for the overlapping regions between grey mouse lemur genome and human genome using ART simulator[@doi:10.1093/bioinformatics/btr708].
`bedtools getfasta` option with the `.bed` file containing homologus regions between two genomes is used to generate a reference `.fasta` file for ART simulator to produce synthetic reads.
By default ART simulator produces a `.fastq` file with flat nucleotide distribution. For better representation of XR-seq characteristics in simulated dataset, we obtain nucleotide distribution frequencies of both species XR-seq reads.
We applied a scoring function (custom script filter_syn.go) where each nucleotide in the simulated read is scored based on the frequency of that nucleotide to be in that position in real dataset and obtained a total score for each simulated read.
Accordingly, best scoring 10 million reads are selected for both species from the simulated dataset.

### Downstream Analysis

1. Calculating RPKM values of RNA-Seq, XR-Seq, and simulated-SR-Seq for both human and lemur at each overlapping region.

Circos plot for comprehensive summarization of all findings has been generated with `circosPlot.R`[@url:https://github.com/compGenomeLab/lemurRepair] by using R package `circlize`.[@doi:doi.org/10.1093/bioinformatics/btu393]
3. Plotly plots.