## Methods

### Fibroblast cell line derivation

### XR-seq library preparation

### RNA-seq library preparation

### Human and lemur alignment

MUMmer version 3.23 [@doi:10.1186/gb-2004-5-2-r12] was used to align grey mouse lemur genome to human genome with `nucmer` subprogram. 
The alignments were filtered based on criteria taking length and identity into account.
Alignments are listed in “delta format” by default by Nucmer.
For every reference (human) - query (lemur) pair, we kept the alignments which form the longest mutually consistent set. 
Filtering step was performed with `delta-filter` subprogram.
`show-coords` subprogram was used to display summary information such as position, percent identity and other features of each alignment, in Btab format with `-B` and `-rclo` arguements.

We used `generateBED.R` [@url:https://github.com/compGenomeLab/lemurRepair] to write homologous human-lemur regions in a bed file. 
Due to low genome coverage in XR-seq data sets small regions introduce random repair values. 
Therefore, because we are not interested in too short alignments, we filtered out alignments shorter than 400 bp while generating the bed file. 
Additionally, alignments with at percent similarity lower than 80 are filtered out of the list. 
Resulting is a BED file with each homologus region between two genomes. 

This output file contains overlapping regions; one region in either genome might be aligned to more than one region in the other genome. 
To remove multi-mapping regions we used `bedtools intersect` [@doi:10.1093/bioinformatics/btq033] to intersect the file with itself. 
The command line we used was: `bedtools intersect -wo -s -a humanOverlapsLemur_short_noFilter.bed -b humanOverlapsLemur_short_noFilter.bed > human_intersect.txt`. 
We compared this intersected output file to the `bed` file generated previously using `findDupAln.R` [@url:https://github.com/compGenomeLab/lemurRepair] to exclude regions in both genomes that align to multiple regions in other genome.
This resulted in a `bed` file of "one-to-one" homologous regions without multiple alignments for one region. 


### XR-seq analysis

### RNA-seq analysis

### XR-seq simulation



