## Methods

### Establishment of the immortalized mouse lemur fibroblast cell line

Primary fibroblasts were established using methods outlined in Larsen and Harris et al [@doi:10.1186/s12915-017-0439-6]. 
For the current study, we used an immortalized version of these primary mouse lemur fibroblasts. 
This cell line was generated by transfecting passage six primary cells at approximately 60% confluency with 5 µg of human hTERT (hTERT) plasmid DNA per well of a 6-well plate using Lipofectamine® 2000 diluted in Opti-MEM® medium. 
Twenty-four hours post transfection, cells were placed under selection using G418 sulfate at a concentration of 300 µg/mL for two weeks. 
Selected cells were then plated into a 96-well plate and remained under G418 sulfate maintenance at a concentration of 100 µg/mL to establish single cell colonies. 
Approximately 2 weeks post transfection, colonies were observed in 19 of the 96 wells and these cells underwent subsequent propagation and expansion. 

### XR-seq library preparation

### RNA-seq library preparation

### Human and lemur alignment

MUMmer version 3.23 [@doi:10.1186/gb-2004-5-2-r12] was used to align grey mouse lemur genome (Mmur_3.0) to human genome (hg19) with `nucmer` subprogram.
The alignments were filtered based on criteria taking length and identity into account.
Alignments were listed in “delta format” by default by Nucmer.
For every reference (human) - query (lemur) pair, we kept the alignments which form the longest mutually consistent set.
Filtering step was performed with `delta-filter` subprogram.
`show-coords` subprogram was used to display summary information such as position, percent identity and other features of each alignment, in Btab format with `-B` and `-rclo` arguements.

We used `generateBED.R` [@url:https://github.com/compGenomeLab/lemurRepair] to write homologous human-lemur regions in a bed file.
Due to low genome coverage in XR-seq data sets small regions introduce random repair values.
Since we were not interested in too short alignments, we filtered out alignments shorter than 400 bp while generating the bed file.
Additionally, alignments with at percent similarity lower than 80 were filtered out of the list.
Resulting was a BED file with each homologus region between two genomes.

This output file contains overlapping regions; one region in either genome might be aligned to more than one region in the other genome.
To remove multi-mapping regions, we used `bedtools intersect` [@doi:10.1093/bioinformatics/btq033] to intersect the file with itself.
The command line we used was: `bedtools intersect -wo -s -a humanOverlapsLemur_short_noFilter.bed -b humanOverlapsLemur_short_noFilter.bed > human_intersect.txt`.
We compared this intersected output file to the `bed` file generated previously using `findDupAln.R` [@url:https://github.com/compGenomeLab/lemurRepair] to exclude regions in both genomes that align to multiple regions in other genome.
This resulted in a `bed` file of "one-to-one" homologous regions without multiple alignments for one region.


### XR-seq analysis

We trimmed 3’ adapter sequences (TGGAATTCTCGGGTGCCAAGGAACTCCAGTNNNNNNACGATCTCGTATGCCGTCTTCTGCTTG) using Cutadapt [@doi:10.14806/ej.17.1.200].
Bowtie2 version 2.3.4.1 [@doi:10.1038/nmeth.1923] was used to align sequencing reads onto reference genomes with default parameters.
With a two-step conversion process, the `.sam` format was first converted (with SAMtools [@doi:10.1093/bioinformatics/btp352]) to `.bam` format, which was then converted to `.bed` format with BEDTools [@doi:10.1093/bioinformatics/btq033].
BED file was sorted by coordinates.
Read duplicates were removed.

### RNA-seq analysis

Sequencing reads were aligned to reference genome using STAR Aligner 2.6.1a [@doi:10.1093/bioinformatics/bts635] with default parameters.
The aligned reads were converted to `.bed` format with `bedtools bamtobed`.
The resulting `.bed` files were sorted by coordinates and duplicate regions were removed.

### XR-seq simulation

We evaluated the correlation of repair mechanisms with both simulated and real XR-seq datasets.
Simulated datasets were generated for the overlapping regions between grey mouse lemur genome and human genome using ART simulator [@doi:10.1093/bioinformatics/btr708].
We used `bedtools getfasta` program with the `.bed` file containing homologus regions between two organisms to generate a reference `.fasta` file for ART simulator to produce synthetic reads.
By default ART simulator produces a `.fastq` file with flat nucleotide distribution.
For a better representation of XR-seq characteristics in simulated dataset, we obtain nucleotide distribution frequencies of both species XR-seq reads.
We applied a scoring function where each nucleotide in the simulated read was scored based on the frequency of that nucleotide to be in that position in real dataset and obtained a total score for each simulated read.
Accordingly, best scoring 10 million reads were selected for both species from the simulated dataset with using `filter_syn.go` [@url:https://github.com/compGenomeLab/lemurRepair]

### Downstream Analysis

We used `regions_rpkm.go` [@url:https://github.com/compGenomeLab/lemurRepair] to calculate RPKM values of XR-Seq, RNA-Seq, and simulated XR-Seq for each overlapping region.
We filtered out regions without any XR-Seq or RNA-Seq reads.

Circos plot for comprehensive summarization of all findings has been generated with `circosPlot.R` [@url:https://github.com/compGenomeLab/lemurRepair] by using R package `circlize`.[@doi:10.1093/bioinformatics/btu393]

3. Plotly plots.

## Data Access

All raw and processed sequencing data generated in this study have been submitted to the NCBI Gene Expression Omnibus (GEO; https://www.ncbi.nlm.nih.gov/geo/) under accession number GSE??????.